mobile/src/home-view.js

```diff
@@ -98,6 +98,7 @@ export default class HomeView extends Component {
     const isDone = scans && !categories.find(cat =>
       (codesByCategory[cat.id] || {count:0}).count < cat.scansRequired)
     const anyScans = !!scans && !!Object.keys(scans).length
+    const categoriesToShow = categories.filter(cat => cat.scansRequired) || []
```

Comment:
The result of `Array.filter(...)` will never be null or undefined.  If no elements match the function, it will return `[]` so the `|| []` is unnecessary